src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogConfiguration.cs

```diff
@@ -40,14 +40,25 @@ public class DatadogConfiguration
         /// </summary>
         public bool UseSSL { get; set; }
 
-        public DatadogConfiguration() : this(DDUrl, DDPort, true) {
+        /// <summary>
+        /// Use TCP or HTTP.
+        /// </summary>
+        public bool UseTCP { get; set; }
+
+        public DatadogConfiguration() : this(DDUrl, DDPort, true, false) {
         }
 
-        public DatadogConfiguration(string url = DDUrl, int port = DDPort, bool useSSL = true)
+        public DatadogConfiguration(string url = DDUrl, int port = DDPort, bool useSSL = true, bool useTCP = false)
         {
             Url = url;
             Port = port;
             UseSSL = useSSL;
+            UseTCP = useTCP;
+        }
+
+        public override string ToString()
+        {
+            return string.Format("{{ Url: {0}, Port: {1}, UseSSL: {2}, UseTCP: {3} }}", Url, Port, UseSSL, UseTCP);
```

Comment:
do we really need to add the numbers if we use the same order ?

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=netframework-4.7.2

I think we do